<?xml version="1.0"?>
<launch>
   <node name="ohm_path_control_node" pkg="ohm_path_control" type="ohm_path_control_node">
      <param name="pub_name_cmd_vel"       type="string" value="vel/teleop" />
      <param name="sub_name_path"          type="string" value="path" />
      <param name="sub_name_pose"          type="string" value="pose" />
      <param name="config_file_controller" type="string" value="/home/m1ch1/workspace/ros/ohm_autonomy/src/ohm_path_control/config/controller.xml"/>
      <param name="config_file_analyser"   type="string" value="/home/m1ch1/workspace/ros/ohm_autonomy/src/ohm_path_control/config/analyser.xml"/>
   </node>
</launch>